src/WsHandler.js

```diff
@@ -46,29 +48,15 @@ class WsHandler {
      * @param {*} meta
      * @param {*} connectionInfo
      */
-    processData(id, meta, connectionInfo, tenantInfo) {
+    processData(id, dumpPath, connectionInfo) {
         logger.info('[WsHandler] Queue for processing id %s', id);
 
         // Metadata associated with a dump can get large so just select the necessary fields.
         const dumpData = {
-            app: meta.applicationName || 'Undefined',
             clientId: id,
-            conferenceId: meta.confName,
-            conferenceUrl: meta.confID,
-            dumpPath: meta.dumpPath,
+            dumpPath,
             endDate: Date.now(),
-            endpointId: meta.endpointId,
-            startDate: meta.startDate,
-            sessionId: meta.meetingUniqueId,
-            userId: meta.displayName,
-            ampSessionId: meta.sessionId,
-            ampUserId: meta.userId,
-            ampDeviceId: meta.deviceId,
-            statsFormat: connectionInfo.statsFormat,
-            isBreakoutRoom: meta.isBreakoutRoom,
-            breakoutRoomId: meta.roomId,
-            parentStatsSessionId: meta.parentStatsSessionId,
-            ...tenantInfo
+            startDate: connectionInfo.startDate
         };
 
         // Don't process dumps generated by JVB & Jigasi, there should be a more formal process to
```

Comment:
bellow this line we persist connections that are JVB and JIGASI these do not go through the `FeatureExtractor` instead they are just saved directly, so the dumpData is still needed at this point.
Also line 77 calls `this.persistDumpData(dumpData);` I don't see that function defined in this class.